### ISSUE TITLE
Additional placeholder

### ISSUE THREAD

**Issue by denis-ko** [James-Yu/LaTeX-Workshop#168]
It would be useful to have a placeholder for the root directory, e.g. %DIR%...

**Comment by James-Yu** [comment]
Can you please give an example use case of this placeholder?

**Comment by denis-ko** [comment]
Sure. Here is an example of a toolchain that uses this placeholder:

```
    "latex-workshop.latex.toolchain": [
    {
        "command": "pdflatex",
        "args": [
            "-aux-directory=tmp",
            "-interaction=nonstopmode",
            "-file-line-error",
            "%DOC%"
        ]
    }, {
        "command": "bibtex",
        "args": [
            "-include-directory=**%DIR%**",
            "tmp\\%DOCFILE%"
        ]
    }, ...
    ] 
```

**Comment by James-Yu** [comment]
@denis-ko  In your use case I'd suggest `./../` in place of %DIR%. Though I agree that there might be some creative usage of this placeholder we won't know. I will work on this soon.

**Comment by James-Yu** [comment]
Will be released in version `2.6.0 soon(TM).
